### ISSUE TITLE
can´t update to Integration Version 1.5.6

### ISSUE THREAD

**Issue by Bloodpack** [Pirate-Weather/pirate-weather-ha#291]
### Describe the bug

can´t update to Integration Version 1.5.6

### Expected behavior

update to Integration Version 1.5.6

### Actual behavior

HACS not showing the update to version 1.5.6

- i have restarted home assistant
- i have rebooted the host
- i have tried to re-download the integration in HACS (hacs shows integration version 1.5.6)
![HACS I-V 1-5-6](https://github.com/user-attachments/assets/a1d9264d-f10f-40ca-9a3e-4cb53a6d3ad9)

in home assistant it shows version 1.5.5

![I-V 1-5-5](https://github.com/user-attachments/assets/a008d310-57b3-428f-bae3-d95481bc23e0)

in home assistant under `config/custom_components/pirateweather` the file `manifest.json` shows this

```json
{
  "domain": "pirateweather",
  "name": "Pirate Weather",
  "codeowners": [
    "@alexander0042"
  ],
  "config_flow": true,
  "dependencies": [],
  "documentation": "https://github.com/alexander0042/pirate-weather-ha",
  "iot_class": "cloud_polling",
  "issue_tracker": "https://github.com/alexander0042/pirate-weather-ha/issues",
  "requirements": [
    "python-forecastio==1.4.0"
  ],
  "version": "1.5.5"
}
```

### Home Assistant version

2024.8.2

### Integration version

1.5.5

### Troubleshooting steps

- [X] I have updated my Home Assistant installation to the latest version.
- [X] I have updated the Pirate Weather Integration to the latest version.
- [X] I have gone through the [documentation](https://pirateweather.net/en/latest/ha/) before opening this issue.
- [X] I have searched this repository and [API Repository](https://github.com/alexander0042/pirate-weather-ha) to see if the issue has already been reported.
- [X] I have restarted my Home Assistant installation.
- [X] I have queried the API in my browser to confirm the issue is not with the API.
- [X] I have written an informative title.

**Comment by cloneofghosts** [comment]
Sorry about this. I forgot to bump the version in the `manifest.json` file which is why HACS and HA display different version numbers. HACS is correct and you are running 1.5.6.

I've bumped the version number and re-released 1.5.6 so if you re-download the update both should show 1.5.6.

**Comment by Bloodpack** [comment]
> Sorry about this. I forgot to bump the version in the `manifest.json` file which is why HACS and HA display different version numbers. HACS is correct and you are running 1.5.6.
> 
> I've bumped the version number and re-released 1.5.6 so if you re-download the update both should show 1.5.6.

re-download did work...

Thank you for the quick fix and your work👍
